CalligraphySample/src/main/java/io/github/inflationx/calligraphy3/sample/PlaceholderFragment.java

```diff
@@ -23,19 +25,19 @@ public static Fragment getInstance() {
     }
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle state) {
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle state) {
```

Comment:
This project assumes nonnull is the default, please only use nullable annotations as needed